packages/storybook/stories/va-alert-uswds.stories.jsx

```diff
@@ -252,22 +252,26 @@ SignInToStartYourApplication.args = {
   ...defaultArgs,
   headline: (
     <h2 slot="headline">
-      Sign in now to save time and save your work in progress
+      {/* Sign in now to save time and save your work in progress */}
```

Comment:
Can commented out code in this file be removed?

Reply:
done